### PR TITLE
Added new game:- SNAKE

### LINKED ISSUE #4174: [New game]: Snake
### 🎮 Game Request

Game logic and basic description:-This game will create a canvas where players control a snake using arrow keys. The snake grows longer as it eats food, and the game ends if the snake collides with itself or the canvas boundaries.

### Point down the features

Game points.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by 1911aditi**
## PR Description 📜
i have added a new game snake. 
assign me under GSSOC'24

Fixes #4174 

<hr>
 
##4174✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸
snake

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @1911aditi ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: README.md [hunk 1]
```diff
@@ -322,6 +322,7 @@ This repository also provides one such platforms where contributers come over an
 | [Find_the_ball](https://github.com/kunjgit/GameZone/tree/main/Games/Find_the_ball)                  |
 |[Color The Page](https://github.com/kunjgit/GameZone/tree/main/Games/Color_The_Page)|
 |[AquaSort_Game](https://github.com/kunjgit/GameZone/tree/main/Games/AquaSort_Game)   |
+| [Snake](https://github.com/1911aditi/GameZone/tree/cf71e07a9bd67d55607e9ffcd4b9b156fe05d031/Games/snake)                               |
```
Comment (ishita-43) [inline README.md:325]: change the path as other game path shows and only change the end with your game name

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Hoping you will make changes soon!!

File: README.md (hunk 1 above)
Comment (1911aditi) [reply README.md]: can you check now.

**Comment by 1911aditi** [comment]
I have resolved the issue. Could you check it now?

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @1911aditi , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**